schedule/yast/btrfs/btrfs+warnings_x86_64.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: btrfs+warnings
```

Comment:
Why we need create new yaml file with suffix _x86_64?  Should we share same yaml file together with default file contain different suffix _86/s390?